sdcm/utils/decorators.py

```diff
@@ -20,6 +20,7 @@
 
 from functools import wraps, partial, cached_property
 from typing import Optional, Callable
+# from sdcm.utils.csrangehistogram import CSRangeHistogramBuilder, CSRangeHistogramSummary
```

Comment:
why this is in comment ?

Reply:
There are special methods in tester, so no need to import them directly. removed